spec/models/game_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+describe Game do
+  it { should validate_presence_of(:start_time) }
```

Comment:
Would this be considered a validation test or something like a _method test_ (for lack of a better term)

```
it "returns a user's full name as a string" do
    user = User.new(
      first_name: "Bob",
      last_name: "Willson"
    )

    expect(user.name).to eq 'Bob Willson'
end
```